bodhi/server/util.py

```diff
@@ -119,19 +119,6 @@ def build_evr(build):
     return tuple(map(str, (build['epoch'], build['version'], build['release'])))
 
 
-def link(href, text):
```

Comment:
Why is this removed? It is not called anymore?

Reply:
yeah, this one is unused now.